corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -265,8 +265,13 @@ hqDefine('app_manager/js/app_manager', function () {
             _.each(childModules, function (childModule) {
                 var parent = modulesByUid[$(childModule).data('rootmoduleid')];
                 if (!parent) {
-                    // This child module is orphaned, throw it at the end
+                    // This child module is orphaned, throw it at the end and alert user
                     $("ul.appnav-module").append(childModule);
+                    var moduleName = $(childModule).data('modulename'),
+                        alertText = _.template(gettext(
+                        'Menu "<%= moduleName %>" references a parent menu that no longer exists'
+                    ))({moduleName: moduleName});
+                    hqImport('hqwebapp/js/alert_user').alert_user(alertText, "danger");
```

Comment:
When they navigate to the module, there's also a warning on the page.

Reply:
So this'll show a warning on every page in app manager, right? Given that there's a warning on the most relevant page and that there's a build error, I'm not convinced this is needed.

Reply:
That's a good point, I'll kill it.